src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -720,7 +716,7 @@ private void doClientFanoutReplyWindow(
         private final long authorization;
         private final KafkaCacheCursor cursor;
         private final KafkaDeltaType deltaType;
-        private final KafkaAge maximumAge;
+        private final KafkaOffsetType maximumAge;
```

Comment:
Please rename `maximumAge` to `maximumOffset` now that we have changed the type from `KafkaAge` to `KafkaOffsetType`.